src/managers/MessageManager.js

```diff
@@ -113,6 +114,83 @@ class MessageManager extends BaseManager {
    * @returns {?Snowflake}
    */
 
+  /**
+   * Edits a message, even if it's not cached.
+   * @param {MessageResolvable} message The message to edit
+   * @param {StringResolvable|APIMessage} [content] The new content for the message
+   * @param {MessageEditOptions|MessageEmbed} [options] The options to provide
+   * @returns {Promise<Message>}
+   */
+  async edit(message, content, options) {
```

Comment:
Just out of curiosity, if I do `manager.edit('123', { embed: { description: 'Hello world' } })`, what would happen? Will it ~~blend~~ break? If so, shouldn't we instead have managers take in all options in an object?

Reply:
Mobile right now so I cant test directly - looking at the source, this should make it's way through the same path as `Message.edit` and `TextChannel.send`.

That is, `APIMessage.create` to `APIMessage.transformOptions` where it does the `typeof content === object` check.

No preference if we keep this or move to single object as you suggested.

Reply:
CC @iCrawl @kyranet ^

Reply:
I personally prefer a single object, it makes a distinction of the content with the ID, which are not to be confused, take:

```js
messages.edit('841298235918516286', { embed: { description: 'Hello world' } });
```

Versus the direct approach:
```js
message.edit({ embed: { description: 'Hello world' } });
```

For somebody who's used to the latter, using the former might confuse them because of the extra argument.